content/bn/chaos-engineering.md

```diff
@@ -0,0 +1,18 @@
+---
+title: কেওস ইঞ্জিনিয়ারিং (Chaos Engineering)
+status: Completed
+category: ধারণা
+tags: ["security", "", ""]
+---
+
+## এটা কি
+
+ক্যাওস ইঞ্জিনিয়ারিং (Chaos Engineering) বা CE হল [ডিস্ট্রিবিউটেড সিস্টেমে (distributed system)](/distributed-systems/) পরীক্ষা করার শৃঙ্খলা যাতে অশান্ত এবং অপ্রত্যাশিত পরিস্থিতি সহ্য করার জন্য ক্ষমতা তৈরি হয় সিস্টেমে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+[SRE](/site-reliability-engineering/) এবং [DevOps](/devops/) অনুশীলনে, প্রোডাক্টের স্থিতিস্থাপকতা (resiliency) এবং [নির্ভরযোগ্যতা (reliability)](/reliability/) বাড়ানোর কৌশলের উপর ফোকাস করে। পর্যাপ্ত পরিষেবার গুণমান নিশ্চিত করার সময় একটি সিস্টেমের ব্যর্থতা সহ্য করার ক্ষমতা সাধারণত একটি সফ্টওয়্যার ডেভেলপমেন্টের প্রয়োজনীয়তা। এমন বেশ কয়েকটি দিক জড়িত যা একটি অ্যাপ্লিকেশনের আউটেজের দিকে নিয়ে যেতে পারে, যেমন অবকাঠামো (infrastructure), প্ল্যাটফর্ম (platform), বা ([মাইক্রোসার্ভিস-ভিত্তিক (microservices)](/microservices/)) অ্যাপ্লিকেশনের অন্যান্য চলমান অংশ। প্রোডাকশন পরিবেশে নতুন ফিচারগুলির খুব তাড়াতাড়ি একের পর এক স্থাপনের ফলে ডাউনটাইম (downtime) হওয়ার সম্ভাবনা বেড়ে যায় এবং একটি গুরুতর ঘটনাও ঘটতে পারে — যা ব্যবসার জন্য যথেষ্ট পরিণতিপূর্ণ।
```

Comment:
Can you kindly make a more suitable sentence for this?
"পর্যাপ্ত পরিষেবার গুণমান নিশ্চিত করার সময় একটি সিস্টেমের ব্যর্থতা সহ্য করার ক্ষমতা সাধারণত একটি সফ্টওয়্যার ডেভেলপমেন্টের প্রয়োজনীয়তা।"

Reply:
"_পর্যাপ্ত পরিষেবার মান নিশ্চিত করার সময়, একটি সিস্টেমের ব্যর্থতা সহ্য করার ক্ষমতা সাধারণত সফ্টওয়্যার ডেভেলপমেন্টে খুব প্রয়োজনীয়।_"
I have come up with this. Suggest something.